app/controllers/trips/user_trips_controller.rb

```diff
@@ -1,5 +1,25 @@
-class Projects::ProjectUsersController < ApplicationController
+class Trips::UserTripsController < ApplicationController
+    before_action :authenticate_user!
+    before_action :set_trip
```

Comment:
tu też bym dała only: [show etc.]

Reply:
ja bym to zostawił. Użytkownik dla każdej akcji w tym kontrolerze musi być :authenticate_user!, w przeciwnym wypadku dając only łatwo można coś przeoczyć.